modules/battery_manager/php/edit.class.inc

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * This file contains the NDB_Form_media class
+ *
+ * PHP Version 7
+ *
+ * @category Loris
+ * @package  Battery_Manager
+ * @author   Victoria Foing <victoria.foing@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace LORIS\battery_manager;
+
+/**
+ * Creates the Edit form for NDB_Form_battery_manager
+ *
+ * @category Loris
+ * @package  Battery_Manager
+ * @author   Victoria Foing <victoria.foing@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class Edit extends \NDB_Form
+{
+    public $skipTemplate = true;
+
+    /**
+     * Entry point for /battery_manager/edit/
+     * Checks if entry id is provided and otherwise redirects to
+     * /battery_manager/ page
+     *
+     * @return none
+     */
+    function setup()
```

Comment:
please add return type in function declaration

Reply:
is "@return void" correct?

Reply:
or should i change it to "@return bool" and add a "return true" at the end?

Reply:
setup() : void